docs/source/sysadmin_guide/configuring_sawtooth/poet_sgx_enclave_configuration_file.rst

```diff
@@ -49,3 +49,7 @@ The ``poet_enclave_sgx.toml`` configuration file has the following options:
   Identifies the PEM-encoded certificate file that was submitted to Intel in
   order to obtain a SPID. Default: none. Specify the full path to the
   certificate file.
+  This pem file can be created from crt and key:
```

Comment:
Is this missing a newline between the previous paragraph and this one?

I'm not familiar with the `` syntax, is that short-hand for code block?

Reply:
There's no newline. All will be seen as a single paragraph after rendering. Written in separate lines for readability.
You can see the result of my changes by clicking on the "view" button in "Files changed" tab.

Reply:
@vaporos Should I be adding a newline?

Reply:
@vaporos , `` marks in-line code (not a code block).

Reply:
@askmish , I like this helpful information.  I don't think a newline is needed before your change.  However, I suggest some wording and formatting changes so that the information is clear to newbies and matches the format of the rest of this document.

Here's what I suggest:

     This pem file can be created from \`\`cert.crt\`\` and \`\`cert.key\`\` files 
     with the following command:
     
     .. code-block:: console

       $ cat cert.crt cert.key > cert.pem

     Or from \`\`cert.pfx\` with this command:

     .. code-block:: console

       $ openssl pkcs12 -in cert.pfx -out cert.pem -nodes